### PR TITLE
Update The Things Fundamentals - Message Types page

### PR DESCRIPTION
**Pull request by pradeeka7**
#### Summary

Updated and fixed minor issues in the Message Types page.

#### Screenshots

image

#### Changes

- Add Uplink and Downlink Messages section
- Fixed some minor issues found
- Converted list to a table in the Calculating Message Integrity Code section.
- Rearranged some sentences

#### Notes for Reviewers

Please update the following image: Issue already reported.
https://github.com/TheThingsNetwork/docs/issues/468

#### Checklist

- [X] Scope: The referenced issue is addressed, there are no unrelated changes.
- [X] Run Locally: Verified that the docs build using `make server`, posted screenshots, verified external links.
- [X] Style Guidelines: Documentation obeys style guidelines in [CONTRIBUTING](CONTRIBUTING.md).
- [X] Commits: Commit messages follow guidelines in [CONTRIBUTING](CONTRIBUTING.md), there are no fixup commits left.

### CODE REVIEW

File: doc/content/lorawan/message-types/index.md [hunk 1]
```diff
@@ -41,57 +42,57 @@ Table: MAC message types in LoRaWAN 1.0.x and 1.1
    </td>
    <td>Join-request
    </td>
-   <td>Uplink OTAA join-request
+   <td>An uplink message, used by the over-the-air activation (OTAA)procedure
```
Comment (benolayinka) [inline doc/content/lorawan/message-types/index.md]:
```suggestion
   <td>An uplink message, used by the over-the-air activation (OTAA) procedure
```

File: doc/content/lorawan/message-types/index.md [hunk 2]
```diff
@@ -41,57 +42,57 @@ Table: MAC message types in LoRaWAN 1.0.x and 1.1
...
    </td>
   </tr>
   <tr>
    <td>Join-accept
    </td>
    <td>Join-accept
    </td>
-   <td>Downlink OTAA join-accept
+   <td>A downlink message, used by the over-the-air activation (OTAA)procedure
```
Comment (benolayinka) [inline doc/content/lorawan/message-types/index.md]:
```suggestion
   <td>A downlink message, used by the over-the-air activation (OTAA) procedure
```

File: doc/content/lorawan/message-types/index.md [hunk 3]
```diff
@@ -117,25 +118,25 @@ In LoRaWAN 1.1, **three** MAC message types are used by the Over-The-Air-Activat
 
 *   Join-request
 *   Join-accept
-*   Rejoin-request
+*   ReJoin-request
```
Comment (benolayinka) [inline doc/content/lorawan/message-types/index.md]:
```suggestion
*   Rejoin-request
```

File: doc/content/lorawan/message-types/index.md [hunk 4]
```diff
@@ -145,16 +146,16 @@ In LoRaWAN versions **earlier** than 1.0.4 the Join-accept message is generated
    </td>
   </tr>
   <tr>
-   <td>Rejoin-request (for all types)
+   <td>ReJoin-request type 0, 1, and 2
```
Comment (benolayinka) [inline doc/content/lorawan/message-types/index.md]: Rejoin

File: doc/content/lorawan/message-types/index.md [hunk 5]
```diff
@@ -145,16 +146,16 @@ In LoRaWAN versions **earlier** than 1.0.4 the Join-accept message is generated
...
    </td>
    <td>JSEncKey
    </td>
   </tr>
 </table>
 
-#### Rejoin-request
+#### ReJoin-request
```
Comment (benolayinka) [inline doc/content/lorawan/message-types/index.md]: Rejoin

File: doc/content/lorawan/message-types/index.md [hunk 6]
```diff
@@ -145,16 +146,16 @@ In LoRaWAN versions **earlier** than 1.0.4 the Join-accept message is generated
...
 
-The Rejoin-request message is always initiated by an end device and sent to the network server. There are three types of Rejoin-request messages: Type 0, 1, and 2. These message types are used to initialize the new session context for the end device. For the Rejoin-request message, the network replies with a Join-accept message.
+The ReJoin-request message is always initiated by an end device and sent to the Network Server. There are three types of Rejoin-request messages: Type 0, 1, and 2. These message types are used to initialize the new session context for the end device. For the Rejoin-request message, the network replies with a Join-accept message.
```
Comment (benolayinka) [inline doc/content/lorawan/message-types/index.md]: Rejoin

File: doc/content/lorawan/message-types/index.md [hunk 7]
```diff
@@ -213,24 +214,31 @@ The frame header (FHDR) of the MAC payload consists of the following fields.
...
+### Sending MAC Commands in FOpts Field
+
+MAC commands can be piggybacked in the FOpts field of a data message for sending. The total length of the MAC commands MUST NOT exceed 15 bytes.
 
 * In LoRaWAN 1.0.x, these piggybacked MAC commands are always sent unencrypted. 
-* In LoRaWAN 1.1, these piggybacked MAC commands are always sent encrypted with the FOpts field using the NwkSEncKey.
+* In LoRaWAN 1.1, these piggybacked MAC commands are always sent encrypted using the NwkSEncKey.
 
-### Transporting MAC Commands and Application-specific data in FRMPayload field
+### Sending MAC Commands and Application-specific Data in the FRMPayload Field
```
Comment (benolayinka) [inline doc/content/lorawan/message-types/index.md]:
```suggestion
### Sending MAC Commands and Application-specific data in the FRMPayload field
```

**Review by benolayinka — APPROVED** [review]

File: doc/content/lorawan/message-types/index.md (hunk 5 above)
Comment (pradeeka7) [reply doc/content/lorawan/message-types/index.md]: suggestion accepted. but the specifications document uses ReJoin and Rejoin interchangeably.
image

File: doc/content/lorawan/message-types/index.md (hunk 6 above)
Comment (pradeeka7) [reply doc/content/lorawan/message-types/index.md]: done

**Pull request merged**